pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2953,6 +2953,22 @@ let
         };
       };
 
+      vscjava.vscode-gradle = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vscjava";
+          publisher = "vscode-gradle";
+          version = "3.12.2023032100";
+          sha256 = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
```

```suggestion
          sha256 = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
```
This is wrong/empty.

---

Please read https://github.com/NixOS/nixpkgs/pull/199157#discussion_r1145507322, was hoping i can get the hash from ofborg.